src/mp_actors/move.py

```diff
@@ -199,7 +245,12 @@ async def _handle_request(
         else:
             result = result_or_callable
         response = Response(request.id, result, None)
+    except StopAsyncIteration:
+        generators.pop(request.id, None)
+        return
     except Exception as e:
-        pickling_support.install(e)
+        import traceback
+        traceback.print_exc()
+        pickling_support.install()
```

Comment:
Is printing the traceback here necessary, or was this just for debugging?

Reply:
Sorry yeah that was just for debugging